### PR TITLE
Add Polish translation (#8)

### PR DESCRIPTION
**Pull request by krkk**
refference to #8

### CODE REVIEW

File: Baconit/Strings/pl/Resources.resw [hunk 1]
```diff
@@ -0,0 +1,886 @@
...
+    <value>Nie</value>
+  </data>
+  <data name="IsNSFW.Content" xml:space="preserve">
+    <value>Jest NSFW?</value>
+  </data>
+  <data name="IsYNSFW.Content" xml:space="preserve">
+    <value>Yes</value>
+  </data>
+  <data name="LastUpdate.Text" xml:space="preserve">
+    <value>Ustatna Aktualizacja:</value>
```
Comment (TiagoDanin) [inline Baconit/Strings/pl/Resources.resw]: `Ustatna` is correct ?

File: Baconit/Strings/pl/Resources.resw [hunk 2]
```diff
@@ -0,0 +1,886 @@
...
+    <value>Odmówiono</value>
+  </data>
+  <data name="FlipViewCode.Text" xml:space="preserve">
+    <value>Odwróć widok</value>
+  </data>
+  <data name="InboxbackgroundupdatingCode.Text" xml:space="preserve">
+    <value>Aktualizacja skrzynki odbiorczej</value>
+  </data>
+  <data name="LastUpdateCode.Text" xml:space="preserve">
+    <value>Ustatna aktualizacja:</value>
```
Comment (TiagoDanin) [inline Baconit/Strings/pl/Resources.resw]: `Ustatna` is correct ?

File: Baconit/Strings/pl/Resources.resw (hunk 1 above)
Comment (krkk) [reply Baconit/Strings/pl/Resources.resw]: This was supposed to be `Ostatnia`

**Review by TiagoDanin — APPROVED** [review]
LGTM 👍

File: Baconit/Strings/pl/Resources.resw [hunk 3]
```diff
@@ -0,0 +1,886 @@
...
+    <value>User Results</value>
+  </data>
+  <data name="ViewInContext.ButtonText" xml:space="preserve">
+    <value>Zobacz komentarz w kontekście</value>
+  </data>
+  <data name="SendCode.Content" xml:space="preserve">
+    <value>Wyślij</value>
+  </data>
+  <data name="UpdateCode.Content" xml:space="preserve">
+    <value>Zakualizuj</value>
```
Comment (TiagoDanin) [inline Baconit/Strings/pl/Resources.resw]: `Zakualizuj` is correct ?

**Comment by krkk** [comment]
So, can you merge it?

**Comment by TiagoDanin** [comment]
@vitorgrs approval is required :)

**Pull request merged**